storage/badger/operation/common.go

```diff
@@ -65,7 +65,7 @@ func insert(key []byte, entity interface{}) func(*badger.Txn) error {
 			return storage.ErrAlreadyExists
 		}
 		if err != badger.ErrKeyNotFound {
```

Comment:
```suggestion
		if !errors.Is(err, badger.ErrKeyNotFound) {
```

furthermore, we could change: 
https://github.com/onflow/flow-go/blob/35d66d677458e61f01df75eed282eef6d3b80732/storage/badger/operation/common.go#L123
to 
```golang
		if errors.Is(err, badger.ErrKeyNotFound) {
```